tests/v2/validation/snapshot/snapshot.go

```diff
@@ -48,7 +48,7 @@ const (
 	concurrencyDefaultValue      = "10%"
 )
 
-func SnapshotRestore(t *testing.T, client *rancher.Client, clusterName string, upgrade string, strategy bool) {
+func Restore(t *testing.T, client *rancher.Client, clusterName string, upgrade string, strategy bool) {
```

Comment:
This one I don't agree with. Since this is a test func "technically" should we preserve it this way?

Reply:
this comes from a linting check that exports types and functions should not have thename duplicated in the package, as for consumers it reads as `snapshot.SnapshotRestore`. What do you propose instead? I see the following options:
- `snapshot.Restore`
- `snapshot.RestoreSnapshot`
- unexporting the function to become `snapshotRestore`

Reply:
This shouldn't even be exported anyway, that was a mistake. `snapshotRestore` would be best